<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FITSDebayerDialog</class>
 <widget class="QDialog" name="FITSDebayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Debayering</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="xOffsetLabel">
       <property name="text">
        <string>X Offset:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="XOffsetSpin">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="yOffsetLabel">
       <property name="text">
        <string>Y Offset:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="YOffsetSpin">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="statusEdit">
       <property name="text">
        <string>Ready.</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="engineLabel">
       <property name="text">
        <string>Engine:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="engineCombo">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>dc1394</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>openCV</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="methodLabel">
       <property name="text">
        <string>Method:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="filterCombo">
       <item>
        <property name="text">
         <string>RGGB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GBRG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GRBG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BGGR</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="methodCombo">
       <item>
        <property name="text">
         <string>Nearest</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Simple</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BILinear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HQLinear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VNG</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FITSDebayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FITSDebayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
